scheduler/scheduler-server/src/test/java/performancetests/recovery/NodeRecoveryTest.java

```diff
@@ -54,22 +54,22 @@
  * @author ActiveEon Team
  * @since 01/12/17
  */
-@RunWith(Parameterized.class)
-public class NodeRecoveryTest extends BaseRecoveryTest {
+//@RunWith(Parameterized.class)
+public class NodeRecoveryTest extends PeformanceTestBase {
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Add some tests to this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2187)